### PR TITLE
2.4 Release notes

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/reference/release-notes/portworx.md [hunk 1]
```diff
@@ -6,6 +6,23 @@ keywords: portworx, release notes
 series: release-notes
 ---
 
+## 2.4
+
+March 2, 2020
+
+### New Features
+
+*
```
Comment (vilasd) [inline content/reference/release-notes/portworx.md]: Why is this showing up here ??

File: content/reference/release-notes/portworx.md [hunk 2]
```diff
@@ -6,6 +6,23 @@ keywords: portworx, release notes
...
+
+### Fixes
+
+The following issues have been fixed:
+
+|**Issue Number**|**Issue Description**|
+|----|----|
+| PWX-10939| We fixed the way license expiration is calculated. <br/><br/>* the composite license (aka "multi-part" license) that expires on <dayX> combined with "extension" license that starts on <dayX+1> introduced a gap on <dayX> where license was handled as EXPIRED (note: it would automatically turn back on on <dayX+1>) <br/><br/>* we're bridging this gap by programatically extending the licenses duration, so each license starts a "day earlier" and expires a "day later" <br/><br/>* [intl. note] this "date-extension" mechanism was there since px-1.3; it is necessary because Flexera's licenses do not deal with timezones (note: TZ modifies "current time" by +-12hr), but it did not work properly -- this fixes it <br/><br/>In addition, we fix the display of license expirations for composite licenses|
+
 ## 2.3.6
```
Comment (vilasd) [inline content/reference/release-notes/portworx.md]: This should not say regression .... Due to an issue in Portworx with earlier releases Portworx calculates license expiratopm ...

**Comment by vilasd** [comment]
@nathan-px add this bug to release notes PWX-10852 : Improved prometheus metrics For Portworx

File: content/reference/release-notes/portworx.md [hunk 3]
```diff
@@ -6,6 +6,26 @@ keywords: portworx, release notes
...
+March 2, 2020
+
+### New Features
+
+* Introducing PX-Central on-premises: Deploy a one-stop dashboard for managing your Portworx clusters and licenses on your own internal and air-gapped infrastructure.
+
+
+### Fixes
+
+<!-- Do we really need to RN this? The regression occurred on 2.4 and 2.5, which haven't been released yet.
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md]: @vilasd see this comment re that fix. Should it even be included?

**Pull request merged**